foundations/introduction/introduction_to_web_development.md

```diff
@@ -65,7 +65,7 @@ It acknowledges that you need a wide variety of skill sets and languages to reac
 
 The Odin Project is constantly evolving because of people like you who get further along in the curriculum and pay it forward by incrementally improving our content over time.
 
-Once you start to feel comfortable with the tools, start contributing to open source projects, like [contributing to The Odin Project](https://www.theodinproject.com/contributing)! The more you contribute, the more you will learn about what you can do, and the closer you will get to being hireable. These projects will also look great on your resume!
+Once you start to feel comfortable with the tools, start contributing to open-source projects, like [contributing to The Odin Project](https://www.theodinproject.com/contributing)! The more you contribute, the more you will learn about what you can do, and the closer you will get to being hireable. These projects will also look great on your resume!
```

Comment:
```suggestion
Once you start to feel comfortable with the tools, start contributing to open source projects, like [contributing to The Odin Project](https://www.theodinproject.com/contributing)! The more you contribute, the more you will learn about what you can do, and the closer you will get to being hireable. These projects will also look great on your resume!
```
I'm going to ask that this is reverted. `open source` occurs over 20 times throughout the curriculum, and not a single one of them is hyphenated. It'd be strange to introduce hyphenating it here.

It's also kind of up to preference whether you hyphenate it or not, and I wouldn't say the hyphenated version is strictly "correct". Some style guides (like [Google's](https://opensource.google/documentation/reference/comms/style-guide#quick_reference)) avoid hyphenating it, and if it's okay for an org that big, it's definitely okay for us.

Reply:
> I'm going to ask that this is reverted. `open source` occurs over 20 times throughout the curriculum, and not a single one of them is hyphenated. It'd be strange to introduce hyphenating it here.
> 
> It's also kind of up to preference whether you hyphenate it or not, and I wouldn't say the hyphenated version is strictly "correct". Some style guides (like [Google's](https://opensource.google/documentation/reference/comms/style-guide#quick_reference)) avoid hyphenating it, and if it's okay for an org that big, it's definitely okay for us.

Ok @JoshDevHub